server/views.py

```diff
@@ -204,6 +209,9 @@ def show_all_help_request(request, page=1):
         end_date = start_date + datetime.timedelta(days=1)
         filter_options['created_date__range'] = (start_date, end_date)
 
+    if len(city_name) != 0 and '' not in city_name:
+        filter_options['city__name'] = city_name[0]
+
     match_qs = HelpRequest.objects.filter(q_option, **filter_options)
```

Comment:
It seems like there's some code duplication between the filtering in the help request table and volunteer table.
We probably want to unify this in some way - maybe open an issue about this (if one doesn't already exist)?

Reply:
We're header toward major refactoring, this should be addressed when designing the views and export functions.